### PR TITLE
Added Donation Tiers to Light Up Chinatown

### PR DESCRIPTION
**Pull request by danthaman44**
Screen Shot 2020-10-27 at 10 11 40 PM
Screen Shot 2020-10-27 at 10 11 29 PM

### CODE REVIEW

**Review by chrisng93 — APPROVED** [review]

File: src/components/LightUpChinatown/DonationTierItem.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+          </Button>
+          <DonationText>
+            {' '}
+            <b>
+              {t('lightUpChinatown.donationAmount', {
+                donationCount: props.currentDonationCount,
+              })}
+            </b>
+          </DonationText>
+          {props.maxDonationCount !== 0 && (
```
Comment (chrisng93) [inline src/components/LightUpChinatown/DonationTierItem.tsx:53]: can the max donation ever be 0?

**Comment by danthaman44** [comment]
Ughh looks like were completely changing the UI of the donation tiers so might need to scrap this...

File: src/components/LightUpChinatown/DonationTierItem.tsx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  maxDonationCount: number;
+}
+
+const DonationTierItem = (props: Props) => {
+  const { t } = useTranslation();
+  const remainingDonationCount =
+    props.maxDonationCount - props.currentDonationCount;
+
+  return (
+    <React.Fragment>
```
Comment (bvillaroman) [inline src/components/LightUpChinatown/DonationTierItem.tsx:25]: If you're wrapping a component under a single node with styling like you do with Container, theres no need for React.fragment.

File: src/components/LightUpChinatown/DonationTierItem.tsx [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+          </ImageContainer>
+        </ColumnContainer>
+        <ColumnContainer>
+          <CTA>{props.ctaText}</CTA>
+          <Title>{props.titleText}</Title>
+          {props.descriptionItems.map((descriptionItem: any) => (
+            <Description>{descriptionItem}</Description>
+          ))}
+          {/* Spacer to keep text aligned between donations items */}
+          <div style={{ width: '600px' }}></div>
```
Comment (bvillaroman) [inline src/components/LightUpChinatown/DonationTierItem.tsx:39]: you could probably do this better in css, like 
`ColumnContainer = styled.div'
 display: flex;
 justify-content: space-between;
'
`

**Review by bvillaroman — COMMENTED** [review]
I would say its almost complete. Only thing is we should move this whole component into /pages directory since it is its own container/not being reused between other components

**Review by jacoblb64 — APPROVED** [review]
Yeah as you mentioned this might be changing a bit, but nice work regardless!

**Pull request merged**